src/main/java/uk/nhs/tis/trainee/notifications/service/PlacementService.java

```diff
@@ -279,8 +284,8 @@ private void createDirectNotifications(Placement placement,
 
       String jobId = PLACEMENT_ROLLOUT_2024_CORRECTION + "-" + placement.getTisId();
       try {
-        Date sendNow = Date.from(Instant.now());
-        notificationService.scheduleNotification(jobId, jobDataMap, sendNow);
+        Date sendInMinute = Date.from(Instant.now().plus(1, ChronoUnit.MINUTES));
```

Comment:
The logic to avoid resending seems the same as for the other placement notifications, so maybe adding a slight delay will allow it to be handled correctly? Otherwise more investigation needed.

Reply:
I'm willing to give it a shot 😄